strawberry_django/filters.py

```diff
@@ -36,10 +36,13 @@
 
 from .arguments import argument
 from .fields.base import StrawberryDjangoFieldBase
+from .settings import strawberry_django_settings as django_settings
 
 if TYPE_CHECKING:
     from django.db.models import QuerySet
 
+
+settings = django_settings()
```

Comment:
I prefer to have this instantiated where it is used to make sure we get any changes made to settings.

Reply:
ack